.repo-metadata.json

```diff
@@ -10,5 +10,7 @@
     "repo": "googleapis/python-bigquery-migration",
     "distribution_name": "google-cloud-bigquery-migration",
     "api_id": "bigquerymigration.googleapis.com",
-    "requires_billing": true
-  }
\ No newline at end of file
+    "requires_billing": true,
+    "default_version": "v1",
```

Comment:
It looks like this repo just has v2alpha: https://github.com/googleapis/python-bigquery-migration/tree/main/google/cloud

Reply:
Good catch! There is a typo in owlbot.py as well: https://github.com/googleapis/python-bigquery-migration/blob/main/owlbot.py#L23